app/Module/GoogleMapsModule.php

```diff
@@ -1609,8 +1624,8 @@ private function adminPlaceCheck() {
 		$x   = 0;
 		$max = 0;
 		while ($x < $i) {
-			$levels                 = explode(",", $place_list[$x]);
-			$parts                  = count($levels);
+			$levels = explode(I18N::$list_separator, $place_list[$x]);
```

Comment:
The `$list_separator` can only be used to **create** strings from lists.
It cannot be used the other way around.  For example, east asian languages do not use "western" puncutation, and the list separator may be a space or even an empty string.  This usage appears to be an internal (i.e. GEDCOM format) string, so the literal `', '` is probably correct here.